src/cli/install.ps1

```diff
@@ -1,176 +1,220 @@
+    Write-Warning "The baseline variant has limited functionality."
+    Write-Warning "If you later want to install the fully featured variant, run this script again without the `-ForceBaseline` parameter."
+} elseif (!$avx2Support -and !$ForceBaseline) {
+    # We should not automatically install a version with limited functionality without the user knowing that.
+    Write-Warning "Bun requires AVX2 support. However, your CPU does not support the AVX2 instruction set."
+    Write-Warning "To install a variant of Bun that is supported on your CPU, but comes with limited functionality, use the `-ForceBaseline` parameter."
+}
+#endregion
+
+#region Installation Path
+if ($InstallToUserProfile) {
+    $installationPath = "$env:USERPROFILE\.bun"
+} elseif ($env:BUN_INSTALL) {
+    if (!(Test-Path -LiteralPath $env:BUN_INSTALL)) {
+        Write-Error "Path in %BUN_INSTALL% (`"$env:BUN_INSTALL`") is invalid." -ErrorAction Stop
+    }
+    $installationPath = $env:BUN_INSTALL
+    Write-Verbose "Found %BUN_INSTALL%."
+} else {
+    $installationPath = "$env:LOCALAPPDATA\Programs\bun"
+}
+Write-Verbose "Installation path set to `"$installationPath`""
+
+# Create installation directory if valid, otherwise throw
+if (!(Test-Path -LiteralPath $installationPath -PathType Container)) {
+    Write-Verbose "Installation directory does not exist."
+    try {
+        Write-Verbose "Create installation directory at `"$installationPath`""
+        New-Item $installationPath -Type Directory | Out-Null
+    } catch {
+        $errorMessage = $PSItem.Exception.Message
+        Write-Error -Message "Error with environment variable BUN_INSTALL. $errorMessage" -Exception $PSItem.Exception -ErrorAction Stop
+    }
+}
+
+# Add or update environment variable
+Write-Verbose "Add user environment variable BUN_HOME=$installationPath"
+[Environment]::SetEnvironmentVariable("BUN_HOME", $installationPath, "User")
+$env:BUN_HOME = $installationPath
+if ([Environment]::GetEnvironmentVariable("PATH", "User") -notlike "*%BUN_HOME%*") {
+    $oldUserPATH = [Environment]::GetEnvironmentVariable("PATH", "User")
+    $newUserPATH = "$oldUserPATH;%BUN_HOME%".Replace(";;", ";")
+    [Environment]::SetEnvironmentVariable("PATH", $newUserPATH, "User")
+}
+$env:Path = [Environment]::ExpandEnvironmentVariables(
+    [Environment]::GetEnvironmentVariable("Path", "Machine") + 
+    ";" + 
+    [Environment]::GetEnvironmentVariable("Path", "User")
+)
+#endregion
+
+#region Bun Download
+$installer = New-TemporaryFile
+
+if ($ForceBaseline) {
+    $file = "bun-windows-x64-baseline.zip"
+} else {
+    $file = "bun-windows-x64.zip"
+}
+
+if ($PSCmdlet.ParameterSetName -eq "Release") {
+    $url = "https://github.com/oven-sh/bun/releases/download/$($Release.ToLower())/$file"
+} elseif ($PSCmdlet.ParameterSetName -eq "Version") {
+    $url = "https://github.com/oven-sh/bun/releases/download/bun-v$Version/$file"
+}
+
+try {
+    Invoke-WebRequest $url -OutFile $installer
```

Comment:
IWR is much slower than curl.exe on Powershell 5. it makes the installation experience worse.

Reply:
I cannot reproduce this

![grafik](https://github.com/oven-sh/bun/assets/2971735/ab5f812e-d3e9-45fa-bcb4-e87dfb2ccbbc)